extensions/amp-ad-exit/0.1/config.js

```diff
@@ -37,7 +37,16 @@ export let AmpAdExitConfig;
 export let NavigationTargetConfig;
 
 /**
- * @typedef {!Object<string, {defaultValue: (string|number|boolean)}>}
+ * @typedef {{
+ *   defaultValue: (string|number|boolean),
+ *   vendorAnalyticsSource: (string|undefined),
+ *   vendorAnalyticsResponseKey: (string|undefined)
+ * }}
```

Comment:
I'd prefer to use this typedef in amp-ad-exit.js. The new properties don't need to be defined by the user, so string|undefined seems appropriate. If the check on L118 was `if (vals.vendorAnalyticsSource && vals.vendorAnalyticsResponseKey)` then the compiler should infer that it's not undefined when used as a key.